### PR TITLE
Editorial: fixed typos in Permissions Policy section

### PR DESCRIPTION
**Pull request by fselcukcan**
***
<a href="https://pr-preview.s3.amazonaws.com/fselcukcan/permissions/pull/290.html" title="Last updated on Aug 31, 2021, 11:31 PM UTC (ca27c30)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/permissions/290/6cac33f...fselcukcan:ca27c30.html" title="Last updated on Aug 31, 2021, 11:31 PM UTC (ca27c30)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1150,15 +1150,15 @@ <h2 id="relationship-to-permissions-policy">
         a HTTP header or a the [^iframe/allow^] attribute). The APIs and features in scope for the
         Permissions Policy specification go beyond those identified in this specification's
         {{PermissionName}} enum (e.g., "sync-xhr" and "gamepad"). In that sense, the Permissions
-        Policy subsumes this specification in that Permissions Policy governs wether a feature is
+        Policy subsumes this specification in that Permissions Policy governs weather a feature is
```
Comment (miketaylr) [inline index.html:1153]: Almost! :)

The correct spelling is "whether" in this instance.

**Review by miketaylr — CHANGES_REQUESTED** [review]
Thanks! Just one typo fix then we're good to merge.

File: index.html (hunk 1 above)
Comment (marcoscaceres) [inline index.html:1153]: English is terrible. 

```suggestion
        Policy subsumes this specification in that Permissions Policy governs whether a feature is
```

**Review by marcoscaceres — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (fselcukcan) [reply index.html]: Oh, :D I made a typo while fixing a typo. Sorry about that. I will laugh awhile at myself, so I am a bit out now.

**Pull request merged**